test/gui/shared/scripts/helpers/api/webdav_helper.py

```diff
@@ -31,3 +32,20 @@ def resource_exists(user, resource):
 def get_file_content(user, resource):
     response = request.get(get_resource_path(user, resource), user=user)
     return response.text
+
+
+def get_folder_items_count(user, folder_name):
+    path = get_resource_path(user, folder_name)
+    xml_response = request.propfind(path, user=user)
+    total_items = 0
+    root_element = ET.fromstring(xml_response.content)
+    for response_element in root_element:
+        for href_element in response_element:
+            #             First href_element is the root folder itself so counting only the href_element of items excluding the href_element of root folder
+            if (
+                href_element.tag == '{DAV:}href'
+                and not href_element.text
+                == f'/remote.php/dav/files/Alice/{folder_name}/'
```

Comment:
something like this should be enough. Use the appropriate method for that in python
```suggestion
                and not href_element.text.endsWith(f'{user}/{folder_name}/')
```